README.md

```diff
@@ -113,4 +113,62 @@ Rather than using a local kafka instance, this project can utilize an instance o
 There is a provided docker-compose file (docker-compose-confluent-cloud.yml) that passes the above environment variables into the container that gets created. Further, this file doesn't spin up a local kafka instance since it is not required.
 
 ### Note
-This has only been tested with Confluent Cloud but technically all SASL authenticated Kafka brokers can be reached using this method.
\ No newline at end of file
+This has only been tested with Confluent Cloud but technically all SASL authenticated Kafka brokers can be reached using this method.
+
+## GitHub Artifact Usage
+
+To use this library in another application, add the GitHub package URLs to the `repositories` section in `pom.xml` of the consumer application or in your local `~/.m2/settings.xml` file. Here is an example implementation of using the GitHub artifact in a consumer application:
+
+```xml
```

Comment:
Are we adding this <settings> xml in either the pom.xml or ~/.m2/settings.xml? Then add the following <dependencies> xml in pom.xml or build.gradle to allow the client application find this library? Could we add a bit detail why doing this below would work? What is the difference between adding <settings> xml to pom.xml and settings.xml?

Reply:
The options to authenticate to [Github Maven Artifacts](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry#authenticating-to-github-packages) are to include the repository along with a read token in the `pom.xml` source code itself or in the supplied `settings.xml` config supplied to the build process. In general, the default `settings.xml` file is found in the user's home directory `~/.m2/settings.xml` but it can also be included as a build argument.

The method that the client application uses to obtain the library jar file is completely up to the client. I have done a few tests with the [jpo-geojsonconverter](https://github.com/usdot-jpo-ode/jpo-geojsonconverter/tree/testing-github-artifacts) application, and I am leaning toward using a supplemental `settings.xml` file to the build process. It would change the maven build from:

```bash
mvn clean install -DskipTests
```
to:

```bash
mvn -s settings.xml clean install -DskipTests
```

The only difference is where the configuration is applied. I still need to do some testing to figure out the best method to implement this with the docker build pipeline while also keeping the access token secret.